extensions/nba-game-viewer/package.json

```diff
@@ -18,62 +18,42 @@
     {
       "name": "upcoming-games",
       "title": "View Upcoming Games",
-      "subtitle": "NBA",
-      "description": "Fetches upcoming NBA games",
+      "subtitle": "NBA & WNBA",
+      "description": "Fetches upcoming NBA or WNBA games",
       "mode": "view"
     },
     {
       "name": "standings",
       "title": "View Standings",
-      "subtitle": "NBA",
-      "description": "Fetches NBA standings",
+      "subtitle": "NBA & WNBA",
+      "description": "Fetches NBA or WNBA standings",
       "mode": "view"
     },
     {
       "name": "headlines",
       "title": "View Headlines",
-      "subtitle": "NBA",
-      "description": "Fetches NBA league headlines from ESPN",
+      "subtitle": "NBA & WNBA",
+      "description": "Fetches NBA or WNBA league headlines from ESPN",
       "mode": "view"
     }
   ],
-  "preferences": [{
-    "name": "conferences",
-    "description": "Choose Standings default conference",
-    "type": "dropdown",
-    "title": "Conferences",
-    "data": [{
-        "title": "Eastern Conference",
-        "value": "Eastern"
-      }, {
-        "title": "Western Conference",
-        "value": "Western"
-      }, {
-        "title": "League",
-        "value": "League"
-    }],
-    "required": true,
-    "default": "Eastern"
-  }],
```

Comment:
Why are the Preference options gone ?

Reply:
No longer needed as discussed in https://github.com/raycast/extensions/pull/13970#discussion_r1717691873. The conferences are present as part of the dropdown and the last used value would be stored. If you have to switch conferences, you needed to use the dropdown in the existing version anyways - so not much value from keeping the preferences.

Reply:
> you needed to use the dropdown in the existing version anyways - so not much value from keeping the preferences.

False, this PR is supposed to add WNBA datas but your modification breaks an existing feature option: Having the ability to drop to a preferred choice every time opening the view. Yours just open the last view, which makes it less usable.

Reply:
@jamtur01 sorry to distract you from the compromise you offered to default the dropdown to value selected in preference. I think more people are used to conventional behavior with other Raycast extensions of storing the last value. I believe we can persist the old behavior, while also adding the support for conventional behavior; I've updated [the gist](https://gist.github.com/jfkisafk/3e2883968d92bb03c4bfb562076fc2b7). Again, it is not a blocker for me if we totally remove `storeValue` (and `useLastValue` preference) - it just looks a little off, but I can get used to it.

Reply:
@jfkisafk 🤷🏻 I think it makes it unnecessarily complex and unwieldy for little apparent benefit, but I am happy to make the changes if that gets this across the line. I pushed another commit.

Reply:
I am on the same page and I agree this is a hack for little benefit. Thanks for the commit to get it across the line @jamtur01